WORKSPACE

```diff
@@ -274,6 +274,13 @@ http_archive(
     url = "https://github.com/aspect-build/rules_ts/releases/download/v3.0.0/rules_ts-v3.0.0.tar.gz",
 )
 
+http_archive(
```

Comment:
- [ ] Since we recently added update-deps.py for workerd, we should be using that for all new dependencies

Reply:
Great to see a reduction in the number of differences between developing with workerd vs internally. How's it look now?

Reply:
Thank you, LGTM! Didn't look closely at js_file but this should be good to go based on Dan's review

Reply:
Well Dan wrote `js_file.bzl` =) I should probably add him as a coauthor on the commit.